### PR TITLE
migrate SpinnerImage to Mui 5

### PR DESCRIPTION
**Pull request by AriaMoradi**

### CODE REVIEW

File: src/components/reader/Page.tsx [hunk 1]
```diff
@@ -120,17 +103,26 @@ const Page = React.forwardRef((props: IProps, ref: any) => {
...
+
     return (
-        <div ref={ref} style={{ margin: 'auto' }}>
+        <Box ref={ref} sx={{ margin: 'auto' }}>
             <SpinnerImage
                 src={`${src}?useCache=${useCache}`}
                 onImageLoad={onImageLoad}
                 alt={`Page #${index}`}
                 imgRef={imgRef}
-                spinnerClassName={`${classes.image} ${classes.loadingImage}`}
```
Comment (AriaMoradi) [inline src/components/reader/Page.tsx:130]: Merging the two might have been wrong in the first place?

**Comment by AriaMoradi** [comment]
semi-approved

**Review by ntbm — COMMENTED** [review]
That's a complicated one.
The Page Spinner seems to be okay, but before the Page numbers are loaded the Spinner is window centered in the Old Version, while being at the same Position as the Page Spinner in the new Version

**Review by ntbm — APPROVED** [review]

**Comment by AriaMoradi** [comment]
So does it need extra fixing?

**Review by ntbm — APPROVED** [review]
For me personally it's a non issue. The other loading indicator looks _nicer_. But replaced fast by the page spinner indicator so I approve.

**Comment by AriaMoradi** [comment]
Lets sleep on this one a bit

File: src/components/reader/Page.tsx (hunk 1 above)
Comment (abhijeetChawla) [reply src/components/reader/Page.tsx:130]: Probably yes, cause they both might have conflicting css. Having different styling objects for both makes more sense

**Review by ntbm — APPROVED** [review]
Looked at it again. Is fine for now considering the due rewrite of the reader.

**Pull request merged**